deploy/crds/infra.watch_serviceassurances_crd.yaml

```diff
@@ -42,6 +42,12 @@ spec:
             eventsEnabled:
               description: Whether the Service Assurance Operator should enable components related to events collection and storage.
               type: boolean
+            haEnabled:
```

Comment:
Is this file still in use now that we use OLM? I think maybe it should be deleted from the tree, but not 100% certain

Reply:
There is some duplication, but I don't know what is required or not. Perhaps we should get @leifmadsen to weight in on this

Reply:
This file is 100% required. When you build the `olm-catalog` with the `operator-courier` tool, it uses the files in `deploy/crds/` and `deploy/cr` to construct the CSV.

Reply:
I would call this out to `highAvailabilityEnabled` or similar.